time stamp
bool data1
bool data2

